### ISSUE TITLE
Cannot index into a null array Object[] with ConvertTo-HPOVPowerShellScript and Get-HPOVNetwork

### ISSUE THREAD

**Issue by tdovan** [HewlettPackard/POSH-HPOneView#485]
### Expected Behavior
I'm trying to export/import HPOVNetwork using ConvertTo-HPOVPowerShellScript and Get-HPOVNetwork but the network are not created in oneview. Bug seems to be in the parsing of ConvertTo-HPOVPowerShellScript which provides an array ( $mBandwidthSystem.Object[]) which is not known by New-HPOVNetwork

```powreshell
New-HPOVNetwork -Name $name -Type $Type -PrivateNetwork $PLAN -SmartLink $smartLink -VLANType $VLANType -VlanID $VLANID -TypicalBandwidth $pBandwidth -MaximumBandwidth $mBandwidthSystem.Object[] -purpose 
```

### Actual Behavior
```powershell
./HPOVNetwork-default.ps1
ParserError: /root/workspace/synergy-config/synergy.obs.hpecic.net/HPOVNetwork-default.ps1:11
Line |
  11 |  … width $pBandwidth -MaximumBandwidth $mBandwidthSystem.Object[] -purpo …
     |                                                                 ~
     | Array index expression is missing or not valid.
```

### Steps to reproduce
```PowerShell
Get-HPOVNetwork -ApplianceConnection $ov2import | ConvertTo-HPOVPowerShellScript -Export HPOVNetwork-default.ps1 –Append
more HPOVNetwork-default.ps1
# -------------- Attributes for Ethernet network "Cohesisty Consumer"
$name                               = "Cohesisty Consumer"
$type                               = "ethernet"
$vLANType                           = "Tagged"
$vLANid                             = 368
$pBandwidth                         = 2500
$mBandwidth                         = 20000
$PLAN                               = $False
$smartLink                          = $True
$purpose                            = "General"
New-HPOVNetwork -Name $name -Type $Type -PrivateNetwork $PLAN -SmartLink $smartLink -VLANType $VLANType -VlanID $VLANID -TypicalBandwidth $pBandwidth -MaximumBandwidth $mBandwidthSystem.Object[] -purpose $purpose

```

### Version Information

```PowerShell
ApplianceVersion: 5.20.01.420365.00   5.20.2422.3962 /root/.local/share/powershell/Modules/HPOneView.520/5.20.2422.3962

Name                           Value
----                           -----
PSVersion                      7.0.2
PSEdition                      Core
GitCommitId                    7.0.2
OS                             Linux 4.18.0-147.8.1.el8_1.x86_64 #1 SMP Thu Apr 9 13:49:54 UTC 2020
Platform                       Unix
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0

```

**Comment by ChrisLynchHPE** [comment]
This is a small bug caused by the parsing if a subnet is associated with the network to then generate the final code.  This will be fixed in the next library update.

**Comment by ChrisLynchHPE** [comment]
This is fixed in [Release 5.20.2470.2147](https://github.com/HewlettPackard/POSH-HPOneView/releases/tag/v5.20.2470.2147) and in [PowerShell Gallery](https://www.powershellgallery.com/packages/HPOneView.520/5.20.2470.2147).

Please verify before this issue is closed.

**Comment by ChrisLynchHPE** [comment]
Closing due to no further activity.  If you wish, feel free to re-open to discuss further.
